crates/wallet/src/wallet/tx_builder.rs

```diff
@@ -295,7 +295,9 @@ impl<'a, Cs> TxBuilder<'a, Cs> {
 
             for utxo in utxos {
                 let descriptor = wallet.get_descriptor_for_keychain(utxo.keychain);
-                let satisfaction_weight = descriptor.max_weight_to_satisfy().unwrap();
+
+                let satisfaction_weight =
+                    descriptor.max_weight_to_satisfy().unwrap().to_wu() as usize;
```

Comment:
Okay we need to create a ticket for what we should do when a caller creates a wallet with descriptors that are insatiable. Panicing with no clear reason when they try construct a tx is not the right behavior imo. Out of scope for this PR.

Reply:
I agree, I tried to solve that problem in #1468 but it needs further discussion.

Reply:
I moved #1468 to ready to review now, we can move this discussion there then.

Reply:
Maybe upstream could add a field to the `Params` structure which would refuse to parse un-satiable descriptors. Could you file an issue in rust-miniscript asking for strategies to handle the error return of max_weight_to_satisfy?

Alternately you can just use `unwrap_or(usize::MAX)` or something silly.

Reply:
Yes, I did it https://github.com/rust-bitcoin/rust-miniscript/issues/695